deploy/db/tinkerbell-init.sql

```diff
@@ -13,7 +13,7 @@ CREATE INDEX IF NOT EXISTS idxgin_type ON hardware USING GIN (data JSONB_PATH_OP
 
 CREATE TABLE IF NOT EXISTS template (
         id UUID UNIQUE NOT NULL
-        , name VARCHAR(200) UNIQUE NOT NULL
+        , name VARCHAR(200) NOT NULL
```

Comment:
This is not used anymore. We use migrations now

https://github.com/tinkerbell/tink/tree/master/db/migration

You have to create a new migration file that ALTERS the current table

Reply:
Done